NimbleSurvey/Sources/Presentation/Modules/SplashScreen/SplashScreen.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  SplashScreen.swift
+//  NimbleSurvey
+//
+//  Created by David Bui on 06/05/2022.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+import SwiftUI
+
+struct SplashScreen: View {
+    var body: some View {
```

Comment:
Please add a line break after the main struct block. To match our convention https://nimblehq.co/compass/development/code-conventions/swift/
```suggestion

    var body: some View {
```

Reply:
Sure, I will